src/recording/recordingmanager.cpp

```diff
@@ -145,26 +172,42 @@ QString& RecordingManager::getRecordingDir() {
 }
 
 // Only called when recording is active.
-void RecordingManager::slotDurationRecorded(QString durationStr)
+void RecordingManager::slotDurationRecorded(quint64 duration)
 {
-    if(m_durationRecorded != durationStr)
+    if(m_secondsRecordedSplit != duration)
     {
-        m_durationRecorded = durationStr;
-        emit(durationRecorded(m_durationRecorded));
+        m_secondsRecordedSplit = duration;
+        if(duration >= m_split_time)
```

Comment:
I think I didn't see on launchpad the bug about the imprecisions with the track start positions. It was on my plan to fix that too, but in the end I couldn't get how to work it out. 

Later on I thought that maybe we don't need to be precise. It is enough with being ahead instead of too late.
That would fix the 00:00:44 that you say, as well as the first song start, which always starts too late. In fact, for the first song, I was thinking on avoiding the counter in metaDataHasChanged until we get a track from getCurrentPlayingTrack for the first time.
So, with that, and assuming that the change took place between the previous call and now, we coud set the time to that of the previous call.

An improved PlayerInfo could help to determine this better, but we also want to avoid filling the cue with several changes forth and back when moving the faders/crossfader as an effect. I guess that's part of the reason why the counter was added initially, although it's not a perfect solution.

As for the warniings, I've just changed that and will commit it later on. Compiling on windows it's quite difficult to get anything out of the console output (except when it directly fails).